data/pigui/modules/station-view/01-lobby.lua

```diff
@@ -228,28 +228,24 @@ end
 
 local function drawPlayerInfo()
 	local station = Game.player:GetDockedWith()
+	if not station or not shipDef then return end
 
-	if(not (station and shipDef)) then return end
-
-	local tech_certified
-
-	if station.techLevel == 11 then
-		tech_certified = l.TECH_CERTIFIED_MILITARY
-	else
-		tech_certified = string.interp(l.TECH_CERTIFIED, { tech_level = station.techLevel})
-	end
-
-	local station_docks = string.interp(l.STATION_DOCKS, { total_docking_pads = string.format("%d", station.numDocks),})
+	local substrings = { shipbay = station:GetAssignedBayNumber(Game.player),
```

Comment:
The bays go from 1 up so I think this should probably be:
`shipbay = station:GetAssignedBayNumber(Game.player) + 1`